mesheryctl/pkg/utils/helpers.go

```diff
@@ -485,3 +502,70 @@ func AskForConfirmation(s string) bool {
 		}
 	}
 }
+
+// CreateConfigFile creates config file in Meshery Folder
+func CreateConfigFile() error {
+	if _, err := os.Stat(fmt.Sprintf("%s/%s", MesheryFolder, "config.yaml")); os.IsNotExist(err) {
+		_, err := os.Create(fmt.Sprintf("%s/%s", MesheryFolder, "config.yaml"))
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+// AddContextToConfig adds context passed to it to mesheryctl config file
+func AddContextToConfig(contextName string, context models.Context, configPath string, set bool) error {
+	var currentConfig models.MesheryCtlConfig
+	if _, err := os.Stat(configPath); os.IsNotExist(err) {
+		return err
+	}
+
+	file, err := ioutil.ReadFile(configPath)
```

Comment:
Use viper here instead?

Reply:
Actually don't need any viper functionality here so didn't use it

Reply:
Got it. This concerns me though in that lack of consistently breeds bugs.